terraform/09-video-hearing-core/20-main.tf

```diff
@@ -134,6 +134,12 @@ module "KeyVault_Secrets" {
       tags         = local.common_tags
       content_type = "secret"
     },
+    {
+      name         = "notifyconfiguration--callbacksecret"
```

Comment:
for the PR pipeline you can get the secret from here https://dev.azure.com/hmcts/Video%20Hearings/_library?itemType=VariableGroups&view=VariableGroupView&variableGroupId=316&path=govuk-notify-dev

for the flux release, this is dependent on [VIH-9086](https://tools.hmcts.net/jira/browse/VIH-9086)
for now I would manually add a secret as a temp

Reply:
Yeah but them values look the same as what is in the current implementation.. after discussion with Murali earlier, new values need to be created for SDS.

would rather leave this in terraform and just update/recreate the secret when we have the correct values.